### ISSUE TITLE
Culling of Stratholme: Several Issues

### ISSUE THREAD

**Issue by Dibbarra** [dalaranwow/dalaran-wow#5257]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
1. Chromie dialogue to skip RP events does not work at all.
2. Unless you have completed Chromie's quest, talking to Arthas will not trigger the RP event properly. (he will not allow one to start the dialogue event).
**Expected behaviour**: Tell us what should happen instead
1. Chromie should let you skip RP.
2. Arthas should not sit and do nothing after talking to him.
**Steps to reproduce the problem**:

1. Try and talk to Chromie.
2. Talk to Arthas without having done Chromie's quest.
3. Watch group fall apart as soon as they load into the dungeon.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Jessica-W** [comment]
Damn am I sick of doing that RP event every time...

**Comment by lkrocek** [comment]
> Culling of Stratholme
> Players may now skip the initial introduction dialog to this dungeon once they have completed it at least once.

http://us.battle.net/wow/en/game/patch-notes/3-3-3

**Comment by Lothloryen** [comment]
Hello there,

Fixed the issue with the event getting stuck, it won't happen anymore. I will keep this issue updated with the rest, I did some rework to this dungeon and might be able to deploy it soon.

**Comment by Lothloryen** [comment]
Hello again,

The whole dungeon was reworked. E.g: Lordaeron Crier announcing the waves, map marks, roleplays inside, encounter improvements, Arthas' escort pathing (smoother, faster...etc). Of course, players will now be able to skip the Intro. It will be deployed to the live realm with the next scheduled maintenance.

Thank you for the report.

**Comment by Lothloryen** [comment]
Already on live.

**Comment by Dibbarra** [comment]
A few minor bugs introduced after this fix: Mal'ganis does not stop attacking after the end event and the first set of infinite dragons don't spawn since the citizens turn into zombies. (Making the proof of demise quest impossible to complete).

**Comment by Lothloryen** [comment]
Thank you for the heads up, Mal'ganis should be fixed already on Algalon. I will take a look into the Infinite Dragons spawn.

**Comment by Lothloryen** [comment]
Hello again,

I do not seem able to reproduce it, can anyone else confirm that the first set of Citizens inside the Town Hall didn't properly transform into the Infinite Dragonflight?

**Comment by RandomXD** [comment]
they spawned and transformed into dragons but just disappeared after and we continued with the next pack

**Comment by lkrocek** [comment]
> daily quests (normal and hc) shouldn't be avaible in 3.3.5

true but when dalaran is progressive server make sense they are up, more sense should be not available for players those are attuned for ICC but in this case attunes should be same for 10m and 25m, so only one type of instance will be required to continue. There is a lot of players those have attunes for icc25m only or reversely. Could be united by this just player reached the patch level, but this is extra ticket.

**Comment by RandomXD** [comment]
did cos again today the issue with the first dragon pack is that after the humans transform into dragons they run to attack you but disappear for no reason, also on the last boss we get no emblems from the chest and mal'ganis keeps using his spells after we kill him during those 10 seconds of roleplay before he's gone

**Comment by Lothloryen** [comment]
Hello there,

Mal'ganis' issue should be fine after next maintenance.
